bin/jobs_functions.js

```diff
@@ -114,6 +122,36 @@ function terminateTestJob(testjobId) {
         })
 }
 
+function deleteTestJob(testjobId) {
+    let url = config.server + '/api/v1/testjobs'+ '/' + testjobId + '/delete';
+    util.output('Deleting job: '+testjobId);
+    request.delete(url)
+        .auth(config.username, config.password)
+        .accept('application/junit+xml')
```

Comment:
This accept looks weird to me, I think it could be application/json, since json is what we return when there is an error. On success we don't return any data. Same goes for cancel(which I know you havent changed but I think it can be fixed too).